app/src/business/calculators/swapCalculatorPMTP.ts

```diff
@@ -394,12 +407,28 @@ export function useSwapCalculator(input: {
       return SwapState.INSUFFICIENT_LIQUIDITY;
     }
 
-    // slippage > 50% can be social engineered as a non-option as to prevent traders from transacting without understanding the potential price volatility before their transaction will actually execute
-    // user entering a negative slippage is not useful but hopefully works out for them
+    if (
+      fromField.value.asset?.symbol === "rowan" &&
+      fromField.value.fieldAmount.greaterThanOrEqual(
```

Comment:
@alanrsoares it should actually go though if the sold amount of rowan is equal to `currentRowanLiquidityThreshold`

also a side note `currentRowanLiquidityThreshold` includes the 18 decimals something to be aware of when comparing.

Reply:
the decimals are being converted here @canercandan. Here I'm checking if the value the user is trying to swap from (selling) is `>=` the current threshold

Reply:
@alanrsoares yep but LIQP will actually work if the amount of rowan sold is less or equal to the threshold.